src/test/ui/lint/outer-forbid.stderr

```diff
@@ -7,23 +7,5 @@ error[E0453]: allow(unused) overruled by outer forbid(unused)
 19 | #[allow(unused, unused_variables, bad_style)]
    |         ^^^^^^ overruled by previous forbid
 
-error[E0453]: allow(unused_variables) overruled by outer forbid(unused)
-  --> $DIR/outer-forbid.rs:19:17
-   |
-17 | #![forbid(unused, non_snake_case)]
-   |           ------ `forbid` level set here
-18 | 
-19 | #[allow(unused, unused_variables, bad_style)]
-   |                 ^^^^^^^^^^^^^^^^ overruled by previous forbid
-
-error[E0453]: allow(bad_style) overruled by outer forbid(non_snake_case)
-  --> $DIR/outer-forbid.rs:19:35
-   |
-17 | #![forbid(unused, non_snake_case)]
-   |                   -------------- `forbid` level set here
-18 | 
-19 | #[allow(unused, unused_variables, bad_style)]
-   |                                   ^^^^^^^^^ overruled by previous forbid
-
-error: aborting due to 3 previous errors
```

Comment:
This is [currently intentioanl](https://github.com/alexcrichton/rust/blob/428b653d28adda89f2ea3f62a24402eddf0961b6/src/librustc/lint/levels.rs#L270-L271) and I felt it wasn't really a "big enough" regression to investigate a fix for.